Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -201,12 +202,26 @@ define([
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showRenderingStatistics', 'Rendering Statistics'));
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showMemoryUsage', 'Memory Usage (MB)'));
 
+        optimizationPanelContents.appendChild(makeCheckbox('skipLevelOfDetail', 'Skip Tile LODs'));
+        var optSSEContainer = document.createElement('div');
+        optSSEContainer.appendChild(makeRangeInput('skipScreenSpaceErrorFactor', 1, 50, 1, 'Skip Screen Space Error Factor'));
+        optimizationPanelContents.appendChild(optSSEContainer);
+        var optBaseScreenSpaceError = document.createElement('div');
+        optBaseScreenSpaceError.appendChild(makeRangeInput('baseScreenSpaceError', 0, 4096, 1, 'Screen space error before skipping LOD'));
+        optimizationPanelContents.appendChild(optBaseScreenSpaceError);
+        var optSkipLevelsContainer = document.createElement('div');
+        optSkipLevelsContainer.appendChild(makeRangeInput('skipLevels', 0, 10, 1, 'Min. levels to skip when loading tiles'));
+        optimizationPanelContents.appendChild(optSkipLevelsContainer);
+        optimizationPanelContents.appendChild(makeCheckbox('immediatelyLoadDesiredLOD', 'Load only tiles that meet the max. screen space error.'));
+        optimizationPanelContents.appendChild(makeCheckbox('loadSiblings', 'Load siblings of visible tiles.'));
```

Comment:
For this it may helps to make the descriptions shorter, even abbreviating "screen space error" as "SSE" would be fine for these.